### PR TITLE
bug: using symbols in matchers causes an error #40

### LINKED ISSUE #40: using symbols in matchers causes an error
```
when(mock).calledWith(Symbol.for(`dummy`));
```
results in "Cannot convert a Symbol value to a string" error at runtime.
The reason is that you try to log debug the matcher using string literals but Symbols can not work in them.
The issue occurs here - 
https://github.com/timkindberg/jest-when/blob/d89f0872a44d3bde4282bf3177d1e1f028cfb1a7/src/when.js#L19

### PR DESCRIPTION
**Pull request by regevbr**
fix #40

### CODE REVIEW

**Comment by regevbr** [comment]
@timkindberg can you please CR?

File: src/when.test.js [hunk 1]
```diff
@@ -223,6 +223,16 @@ describe('When', () => {
       expect(fn(2)).toEqual('y')
     })
 
+    it('should handle symbol matchers', () => {
+      const fn = jest.fn()
+      const symbol = Symbol.for(`sym`)
+      when(fn).calledWith(symbol, 2).mockReturnValue('x')
+
+      expect(fn(5)).toBeUndefined()
```
Comment (timkindberg) [inline src/when.test.js]: Do you want an assertion that is calling the fn and getting 'x' back? Like the happy path.

Comment (regevbr) [reply src/when.test.js]: Sure, I just copied the test below it

Comment (regevbr) [reply src/when.test.js]: actually that was a good catch as it found another similar bug. See updated code

**Comment by timkindberg** [comment]
jest-when@2.7.1

**Pull request merged**